### PR TITLE
Docs nested blockquotes fix

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/contributing/git-commit-message-convention.html))

Docs (block-quote): nested block quotes functionality description.

### CODE REVIEW

**Comment by jacekbogdanski** [comment]
https://ckeditor.com/docs/ckeditor5/latest/framework/guides/deep-dive/schema.html#generic-items
> The side effect of such a definition inheritance is that now <blockQuote> is allowed in <blockQuote> which needs to be resolved by a callback which will disallow this specific structure.

This guide is using real examples for blockquote - nesting is now a feature, so maybe it makes sense to remove or update this sentence?

File: packages/ckeditor5-block-quote/docs/features/block-quote.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ Use the editor below to see the block quote plugin in action. You can also prece
 
 {@snippet features/block-quote}
 
-## Nesting block quotes
+## Nested block quotes
 
-Starting from version 27.1.0, CKEditor 5 supports nesting a block quote in another block quote. This may prove useful for scientific or academic papers, but also articles citing sources and referring to previous writing would often use it. To use this functionality, simply click where you need the quote nested and insert it just like a regular one.
+Starting from version 27.1.0, CKEditor 5 will properly display a block quote nested in another block quote. The functionality itself may prove useful for scientific or academic papers, but also articles citing sources and referring to previous writing would often use it.
+
+This is a backwards compatibility issue for content created in CKEditor 4. Pasting content with nested block quotes is supported. It is also possible to nest a block quote into another block quote using the {@link features/drag-drop drag and drop} mechanism - select an existing blockquote and drag it into another existing blockquote.
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/features/block-quote.md:23]:
```suggestion
Support for nested block quotes is provided as backward compatibility for loading pre-existing content, for example created in CKEditor 4. Additionally, pasting content with nested block quotes is possible, too. You can also nest a block quote in another block quote using the {@link features/drag-drop drag and drop} mechanism &mdash; just select an existing block quote and drag it into another.
```

Using both "block quote" and "blockquote" in one sentence seems pretty extravagant to me :stuck\_out\_tongue\_winking\_eye:

**Comment by AnnaTomanek** [comment]
> This guide is using real examples for blockquote - nesting is now a feature, so maybe it makes sense to remove or update this sentence?

Thanks @jacekbogdanski - can we just remove this sentence? I suppose the callback that disallows this structure is already removed, right?

**Comment by jacekbogdanski** [comment]
From what I see the check for nested blockquote was removed in https://github.com/ckeditor/ckeditor5/pull/9382/files so I suppose we can just get rid of it 👍🏻

**Comment by godai78** [comment]
> I suppose the callback that disallows this structure is already removed, right?

I believe this is exactly what was removed by @psmyrek from the code, if I read it right. We now need to specifically add code to prevent this behaviour, as described in the Block Quote docs.

**Comment by AnnaTomanek** [comment]
Thanks Jacek! @godai78 can you please remove it as part of this PR.

File: packages/ckeditor5-block-quote/docs/features/block-quote.md [hunk 2]
```diff
@@ -16,9 +16,11 @@ Use the editor below to see the block quote plugin in action. You can also prece
 
 {@snippet features/block-quote}
 
-## Nesting block quotes
+## Nested block quotes
 
-Starting from version 27.1.0, CKEditor 5 supports nesting a block quote in another block quote. This may prove useful for scientific or academic papers, but also articles citing sources and referring to previous writing would often use it. To use this functionality, simply click where you need the quote nested and insert it just like a regular one.
+Starting from version 27.1.0, CKEditor 5 will properly display a block quote nested in another block quote.  It would be most useful in email editors. Keeping all the nested quotes, citing previous messages, within a mail communication is crucial to maintain the flow of email exchange. The functionality itself may also prove useful for scientific or academic papers, but articles citing sources and referring to previous writing would often use it, too.
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/features/block-quote.md:21]:
```suggestion
Starting from version 27.1.0, CKEditor 5 will properly display a block quote nested in another block quote. This sort of structure is indispensable in email editors or discussion forums. The ability to cite previous messages and preserve a correct quotation structure is often crucial to maintain the flow of communication. Nested block quotes may also prove useful for scientific or academic papers, but articles citing sources and referring to previous writing would often use it, too.
```

File: packages/ckeditor5-block-quote/docs/features/block-quote.md [hunk 3]
```diff
@@ -16,9 +16,11 @@ Use the editor below to see the block quote plugin in action. You can also prece
...
+
+Support for nested block quotes is provided as backward compatibility for loading pre-existing content, for example created in CKEditor 4. Additionally, pasting content with nested block quotes is supported, too. You can also nest a block quote in another block quote using the {@link features/drag-drop drag and drop} mechanism &mdash; just select an existing block quote and drag it into another.
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/features/block-quote.md:23]:
```suggestion
Support for nested block quotes is provided as backward compatibility for loading pre-existing content, for example created in CKEditor 4. Additionally, pasting content with nested block quotes is supported. You can also nest a block quote in another block quote using the {@link features/drag-drop drag and drop} mechanism &mdash; just select an existing block quote and drag it into another.
```

File: packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html [hunk 4]
```diff
@@ -1,35 +1,26 @@
...
-	<p>I looked and looked but I didn&#39;t see God.</p>
-	</blockquote>
-
-	<p>Let us refer to the <strong>Yuri Gagarin's</strong> <a href="https://en.wikiquote.org/wiki/Yuri_Gagarin#Disputed">Wikiquote page</a> on that topic and dig further into this particular case study, as it offers a plausible and factual explanation of this mess-up:</p>
+		<blockquote>
+		<p>Tom,</p>
 
-	<blockquote>
-	<p>Colonel Valentin Petrov stated in 2006&nbsp;that the cosmonaut never said such words, and that the quote originated from Nikita Khrushchev&#39;s speech at the plenum of the Central Committee of the CPSU about the state&#39;s anti-religion campaign, saying:</p>
+		<p>this is just a reminder, that we need the data for Friday meeting with the external subcontractors. Can you, please, let me know, if all the reports are ready?</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html:8]:
```suggestion
		<p>This is just a reminder that we need the data for our Friday meeting with the external subcontractors. Can you please let me know, if all the reports are ready?</p>
```

File: packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html [hunk 5]
```diff
@@ -1,35 +1,26 @@
...
 
-	<blockquote>Gagarin flew into space, but didn&#39;t see any god there.</blockquote>
+		<p>Jenna</p>
+		</blockquote>
 
-	<p>Gagarin himself was a member of the Russian Orthodox Church.</p>
-	</blockquote>
+	<p>Jenna,</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html:13]:
```suggestion
	<p>Hi Zoe,</p>
```

File: packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html [hunk 6]
```diff
@@ -1,35 +1,26 @@
...
 
-	<p>It looks like a cunning rhetoric to support an ideological point of view and you may even suppose the cosmonaut himself was not personally involved in it. Quite honestly, being a believer, he would have a hard time denying the existence of God. It is something Gherman Titov, the second Soviet cosmonaut, would rather say, being an atheist.</p>
+	<p>the reports are coming in, but the Q3 data is still missing. Far as I can tell, Shizuoka is behind the schedule with the final summary and the Expansion Department has not released their numbers yet. I will reach out to both and will keep you informed on that topic by noon, tomorrow.</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html:15]:
```suggestion
	<p>The reports are coming in, but the Q3 data is still missing. As far as I can tell, Shizuoka is behind the schedule with the final summary and the Expansion Department has not released their numbers yet. I will reach out to both and will keep you informed on that topic by noon, tomorrow.</p>
```

File: packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html [hunk 7]
```diff
@@ -1,35 +1,26 @@
...
-	<p>Let us refer to the <strong>Yuri Gagarin's</strong> <a href="https://en.wikiquote.org/wiki/Yuri_Gagarin#Disputed">Wikiquote page</a> on that topic and dig further into this particular case study, as it offers a plausible and factual explanation of this mess-up:</p>
+		<blockquote>
+		<p>Tom,</p>
 
-	<blockquote>
-	<p>Colonel Valentin Petrov stated in 2006&nbsp;that the cosmonaut never said such words, and that the quote originated from Nikita Khrushchev&#39;s speech at the plenum of the Central Committee of the CPSU about the state&#39;s anti-religion campaign, saying:</p>
+		<p>this is just a reminder, that we need the data for Friday meeting with the external subcontractors. Can you, please, let me know, if all the reports are ready?</p>
 
-	<blockquote>Gagarin flew into space, but didn&#39;t see any god there.</blockquote>
+		<p>Jenna</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html:10]:
```suggestion
		<p>Zoe</p>
```

File: packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html [hunk 8]
```diff
@@ -1,35 +1,26 @@
...
-	<p>It was, in fact, duly noted in <i>The New York Times</i>, dated May 7, 1962:</p>
-	<blockquote>
-		Maj. Gherman S. Titov, the Soviet astronaut, proclaimed his disbelief in God today. He said he saw
-		<blockquote>no God or angels</blockquote>
-		during his seventeen orbits of the earth.
+	<p>Tom</p>
 	</blockquote>
 
-	<p>A similar statement by Titov was also reported by Pavel Barashev and Yuri Dokuchayev, so it seems this was not entirely made up by Khrushchev, but rather reused and falsely attributed to the Earth's first space farer.</p>
+	<p>Please do so, this is getting urgent. Contact Kamalla from ED directly. Also check Shizuoka, this is very much unlike him, to leave such and important matter for the last moment.</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html:20]:
```suggestion
	<p>Please do so, this is getting urgent. Contact Haleema from ED directly. Also check Shizuoka, this is very much unlike him to leave such an important matter for the last moment.</p>
```

File: packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html [hunk 9]
```diff
@@ -1,35 +1,26 @@
...
 
-	<p>Apparently there was more than one source involved in the origin of this misunderstanding. As you can see, sometimes, just like in the case of the notorious &quot;Einstein fish&quot;, things can get messed up pretty easily. And it takes a lot of time to debunk them, as there are numerous occurrences of the quote that <i>partially</i> support the case.</p>
+	<p>Hope to get this ready for review at the 2pm board meeting, if you can manage to prepare an extract and to have the full data by Thursday morning at the last.</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html:22]:
```suggestion
	<p>Hope to have this ready for review at the 2pm board meeting provided you can manage to prepare a summary and to have the full data by Thursday morning at the latest.</p>
```

File: packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html [hunk 10]
```diff
@@ -1,35 +1,26 @@
...
 	</blockquote>
 
-	<p>A similar statement by Titov was also reported by Pavel Barashev and Yuri Dokuchayev, so it seems this was not entirely made up by Khrushchev, but rather reused and falsely attributed to the Earth's first space farer.</p>
+	<p>Please do so, this is getting urgent. Contact Haleema from ED directly. Also check Shizuoka, this is very much unlike him to leave such an important matter for the last moment.</p>
 
-	<p>Apparently there was more than one source involved in the origin of this misunderstanding. As you can see, sometimes, just like in the case of the notorious &quot;Einstein fish&quot;, things can get messed up pretty easily. And it takes a lot of time to debunk them, as there are numerous occurrences of the quote that <i>partially</i> support the case.</p>
+	<p>Hope to have this ready for review at the 2pm board meeting provided you can manage to prepare a summary and to have the full data by Thursday morning at the latest.</p>
 
-	<p>This is why you should always check your sources twice and, if necessary, quote them!</p>
+	<p>Jenna</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html:24]:
```suggestion
	<p>Zoe</p>
```

:woman\_facepalming:

File: packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html [hunk 11]
```diff
@@ -1,35 +1,26 @@
 <div id="snippet-nested-block-quote">
-	<h3>The Gagarin mixup</h3>
-
-	<p>Misquoting is more widespread than one may think. And it may happen to almost anybody. Sometimes it would be an accident, but sometimes you may consider this a conscious act. One of the better known cases is something a Soviet cosmonaut, Yuri Gagarin, was supposed to say &mdash; but apparently never said. The infamous quote goes:</p>
+	<p>Re: material usage information</p>
```
Comment (AnnaTomanek) [inline packages/ckeditor5-block-quote/docs/_snippets/features/nested-block-quote.html:2]:
```suggestion
	<p>Re: Material usage information</p>
```

**Pull request merged**